src/frontend/index.js

```diff
@@ -8,6 +11,12 @@ requirejs.config({
         { name: 'mod_parser', location: '../backend/parser', main: 'index'}
     ],
     paths: {
+        lib: '../bower_components',
+        module: '../backend',
+	jquery: "../bower_components/jquery/dist/jquery.min",
```

Comment:
Dependencies are now handled in "packs". Refer to my pull request for an example. Please be consistent.